Doc/whatsnew/3.13.rst

```diff
@@ -82,69 +105,105 @@ Interpreter improvements:
+  It is currently disabled by default (though we may turn it on later).
+  Performance improvements are modest -- we expect to be improving this
+  over the next few releases.
+* Color support in the new :ref:`interactive interpreter
+  <whatsnew313-better-interactive-interpreter>`,
+  as well as in :ref:`tracebacks <whatsnew313-improved-error-messages>`
+  and :ref:`doctest <whatsnew313-doctest>` output.
+  This can be disabled throughhe :envvar:`PYTHON_COLORS` and |NO_COLOR|_
+  environment variables.
 
-New typing features:
+Python data model improvements:
 
-* :pep:`696`: Type parameters (:data:`typing.TypeVar`, :data:`typing.ParamSpec`,
-  and :data:`typing.TypeVarTuple`) now support defaults.
+* :attr:`~class.__static_attributes__` stores the names of attributes accessed
+  through ``self.X`` in any function in a class body.
+* :attr:`!__firstlineno__` records the first line number of a class definition.
 
-* :pep:`702`: Support for marking deprecations in the type system using the
-  new :func:`warnings.deprecated` decorator.
+Significant improvements in the standard library:
 
-* :pep:`742`: :data:`typing.TypeIs` was added, providing more intuitive
-  type narrowing behavior.
+* Add a new :exc:`PythonFinalizationError` exception, raised when an operation
+  is blocked during :term:`finalization <interpreter shutdown>`.
+* The :mod:`argparse` module now supports deprecating command-line options,
+  positional arguments, and subcommands.
+* The new functions :func:`base64.z85encode` and :func:`base64.z85decode`
+  support encoding and decoding `Z85 data <https://rfc.zeromq.org/spec/32/>`_.
+* The :mod:`copy` module now has a :func:`copy.replace` function,
+  with support for many builtin types and any class defining
+  the :func:`~object.__replace__` method.
+* The :mod:`dbm.sqlite3` module is now the default :mod:`dbm` backend.
+* The :mod:`os` module has a suite of new functions for working with Linux's
+  timer notification file descriptors.
+* The :mod:`random` module now has a :ref:`command-line interface <random-cli>`.
 
-* :pep:`705`: :data:`typing.ReadOnly` was added, to mark an item of a
-  :class:`typing.TypedDict` as read-only for type checkers.
+Security improvements:
 
-Free-threading:
+* :func:`ssl.create_default_context` sets :data:`ssl.VERIFY_X509_PARTIAL_CHAIN`
+  and :data:`ssl.VERIFY_X509_STRICT` as default flags.
 
-* :pep:`703`: CPython 3.13 has experimental support for running with the
-  :term:`global interpreter lock` disabled when built with ``--disable-gil``.
-  See :ref:`Free-threaded CPython <free-threaded-cpython>` for more details.
+C API improvements:
+
+* The :c:data:`Py_mod_gil` slot is now used to indicate that an extension module
+  supports running with the :term:`GIL` disabled.
+* The :doc:`PyTime C API </c-api/time>` has been added,
+  providing access to system clocks.
+* :c:type:`PyMutex` is a new lightweight mutex that occupies a single byte.
```

Comment:
@vstinner / @serhiy-storchaka -- is there anything you want to highlight particularly in terms of C-API changes for the *release highlights* section of What's New? These were my suggestions but very happy to change them.

A

Reply:
Nothing particular, but entries in the "C API Changes" section should be reordered. First new functions, then changes in existing functions. Functions with same prefix (e.g. `PyDict_`) or similar purpose should be mentioned next to one other.

You can highlight adding groups of functions with similar effect: for example adding functions that return  strong references rather than borrowed references or adding functions that return different codes for error and the absent item (without raising an exception in the latter case).